src/canvas/Canvas.ts

```diff
@@ -788,9 +788,9 @@ export class Canvas extends SelectableCanvas implements CanvasOptions {
   __onMouseUp(e: TPointerEvent) {
     const transform = this._currentTransform;
     this._cacheTransformEventData(e);
-    const target = this._target;
     const isClick = this._isClick;
     this._handleEvent(e, 'up:before');
+    const target = this._target;
```

Comment:
at this point let's also move isClick down here?

Reply:
I also want isClick to be available only on mouseup event